Kernel/PolynomialNormalizer.cpp

```diff
@@ -25,13 +25,16 @@ struct RenderMonom {
     using Monom        = Monom       <NumTraits>;
     auto& raw = x.raw();
     std::sort(raw.begin(), raw.end());
-    auto fstNum = Option<Numeral>();
+
```

Comment:
GCC reckons that the value of `fstNum` could be unitialised. I _think_ it's a false-positive, but this version doesn't produce a warning and is clearer in my view.

Reply:
@joe-hauns - meant to tag you in this, any objections?

Reply:
Strange that it gives a false positive there, but yeah looks good!